app/features/settings/actionTypes.js

```diff
@@ -8,6 +8,16 @@
  */
 export const SET_AVATAR_URL = Symbol('SET_AVATAR_URL');
 
+/**
+ * The type of (redux) action that sets the default domain.
+ *
+ * {
+ *     type: SET_DEFAULT_DOMAIN,
+ *     defaultDomain: string
+ * }
+ */
+export const SET_DEFAULT_DOMAIN = Symbol('SET_DEFAULT_DOMAIN');
```

Comment:
let's call this entire thing a "server URL". this is consistent with jitsi meet on mobile.

Reply:
Okey. I'll update this.

Reply:
@saghul 
On second thought, aren't we only storing domain? 
Why should we call this server URL?
Since we can't support HTTP here we don't need full URL just domain would be sufficient.

Reply:
We support deploying in a subdirectory, so something like https://example.com/meet/ should be valid too.

We can strip the scheme if specified, and force it to https, that’s ok.

Reply:
Oh. In that case, I'll update this.